python/rmm/_lib/lib.pyx

```diff
@@ -269,3 +269,37 @@ def rmm_getallocationoffset(ptr, stream):
     result = int(c_offset[0])
     free(c_offset)
     return result
+
+
+cdef memory_pair c_getinfo(
+    cudaStream_t stream
+) except *:
+    """Get total and free memory sizes using the RMM memory manager by calling
+    the librmm functions via Cython
+    """
+    cdef size_t free
+    cdef size_t total
+
+    with nogil:
+        rmm_error = rmmGetInfo(
+            <size_t*>&free,
+            <size_t*>&total,
+            <cudaStream_t>stream
+        )
+
+    check_error(rmm_error)
+
+    return memory_pair(free, total)
+
+
+def rmm_getinfo(stream):
+    """Get total and free memory sizes using the RMM memory manager by calling
+    the librmm functions via Cython
+    """
+    cdef cudaStream_t c_stream = <cudaStream_t><size_t>stream
+
+    cdef memory_pair memory_info = c_getinfo(
+        <cudaStream_t>c_stream
+    )
+
+    return int(memory_info.first), int(memory_info.second)
```

Comment:
I'm unsure what Numba expects for this interface here, but would it be possible / make sense to return a namedtuple here to make it more self-documenting?

Reply:
There isn't really an expectation of an interface at the moment, but internally for a similar purpose Numba uses a `namedtuple` with `free` and `total` as members, so I've made `get_info` return a `namedtuple` with similar fields here.